docs/en/siem/rules-api-delete.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[rules-api-delete]]
+=== Delete rule
+
+Deletes a single rule using the `rule_id` or `id` field.
+
+==== Request URL
+
+`DELETE /api/detection_engine/rules`
+
+===== URL query parameters
+
+The URL query must include one of the following:
+
+* `id` - `DELETE /api/detection_engine/rules?id=<id>`
+* `rule_id`- `DELETE /api/detection_engine/rules?rule_id=<rule_id>`
+
+===== Example request
+
+Deletes the rule with an `id` value of `16947168-5405-453d-a8b5-aadad357af42`:
+
+[source,js]
+--------------------------------------------------
+DELETE api/detection_engine/rules?id=16947168-5405-453d-a8b5-aadad357af42
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
```

Comment:
Sample response for delete. Not sure if we want to include a sample response for delete.

```json
{
  "created_at": "2020-01-29T23:10:13.950Z",
  "updated_at": "2020-01-29T23:12:56.282Z",
  "created_by": "user",
  "description": "Query with a rule_id that acts like an external id",
  "enabled": true,
  "false_positives": [],
  "from": "now-6m",
  "id": "ca9f25aa-ffff-5555-ffff-1ec301a02529",
  "immutable": false,
  "interval": "5m",
  "rule_id": "query-rule-id",
  "language": "kuery",
  "output_index": ".siem-signals-devin-hurley-default",
  "max_signals": 100,
  "risk_score": 1,
  "name": "Changes only the name to this new value",
  "query": "user.name: root or user.name: admin",
  "references": [],
  "severity": "high",
  "updated_by": "user",
  "tags": [],
  "to": "now",
  "type": "query",
  "threat": [],
  "version": 2,
  "status": "succeeded",
  "status_date": "2020-01-29T23:10:19.904Z",
  "last_success_at": "2020-01-29T23:10:19.904Z",
  "last_success_message": "succeeded"
}
```